spdmlib/src/common/mod.rs

```diff
@@ -638,6 +733,15 @@ pub struct SpdmProvisionInfo {
     pub peer_cert_chain_data: Option<SpdmCertChainData>,
     pub peer_cert_chain_root_hash: Option<SpdmDigestStruct>,
     pub default_version: SpdmVersion,
+    pub opaque_data_support: bool,
+    pub opaque_data_fmt0: bool, // Support or not, true for support, if support
+    // this field shall indicate that the format for
+    // all OpaqueData fields in this specification
+    // is defined by the device vendor or other standards body.
+    pub opaque_data_fmt1: bool, // Support or not, true for support, if support
+                                // this field shall indicate that the format for
+                                // all OpaqueData fields in this specification
+                                // is defined by the General opaque data format.
```

Comment:
please treat opaque data format as one type of algo.
No need to provision here.

Reply:
> 

Well, please check the updated one, Thank you!